package.json

```diff
@@ -86,7 +86,7 @@
     "@bbc/psammead-live-label": "^1.0.0",
     "@bbc/psammead-locales": "^4.1.9",
     "@bbc/psammead-media-indicator": "^4.0.7",
-    "@bbc/psammead-media-player": "^2.7.11",
+    "@bbc/psammead-media-player": "^2.7.13",
```

Comment:
Is this solving a separate issue? This PR is marked as resolving https://github.com/bbc/simorgh/issues/6950, which doesn't mention Media Player. It's also closed. I'm confused. 😅  Please can you clarify?

Reply:
@simonsinclair, #6950 just highlights the bug, part of the issue is due to the mediaplayer not having substantial between items below it. that was addressed in https://github.com/bbc/psammead/pull/3583, this pulls that change and also ensures the main column maintains a 56px spacing between it and the footer.

Reply:
Thanks for explaining, Rowland. So:
- A `1.5rem` (24px) bottom margin on Media Player, plus
- `GEL_SPACING_QUAD` (32px) bottom padding on the Story Page

Will equal 56px padding _when_ there is a Media Player at the bottom of the page.

Please can you help me understand what was causing this → https://user-images.githubusercontent.com/52247014/85015341-b7b01080-b15f-11ea-8162-0319fd361ce9.png. If the Media Player's bottom margin was previously `2rem`, why are we not seeing any space in that screenshot?

Reply:
@simonsinclair yeah, so if you visit the original issue that's where [this is apparent](https://user-images.githubusercontent.com/52247014/85015341-b7b01080-b15f-11ea-8162-0319fd361ce9.png), however, 

`
"If the Media Player's bottom margin was previously 2rem, why were we not seeing any space in the screenshot?"
`

It wasn't previously 2rem, there was nothing. [This first pr](https://github.com/bbc/psammead/pull/3575) initiated the 2rem fix, but after further clarification from @lizcameron I had to fix forward with https://github.com/bbc/psammead/pull/3583. 

> And when there is not a Media Player at the bottom of the page, we trust that the last item will also have a 1.5rem (24px) bottom margin?

I can confirm that text and image contents apart from headings always have a 1.5rem bottom padding, so I presume it will always translate to 56px eventually.

Reply:
Thanks, Rowland. 👌